security/rs-security-best-practices.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Best Practices
+page_title: Best Security Practices when Installing and Using Telerik Report Server
+description: "Check the recommended security practices when installing the Telerik Report Server and working with its reports, users, and services."
+slug: rs-security-best-practices
+tags: telerik, reporting, report, server, security, best, practices
+published: True
+position: 4
+---
+
+# Security Best Practices
+
+The article discusses general and report server-specific security practices. While the recommendations may be necessary to keep your Telerik Report Server instance with all report definitions, data connections, etc. secure, they may not be sufficient. The article should not be regarded as a complete and comprehensive security guidance.
+
+## Configuration Settings
+
+The suggestions in this section are the responsibility of the administrator deploying the Telerik Report Server. They are general security settings of web applications deployed on the Windows IIS Servers. We have implemented most of them and let you switch them as configuration options when installing the Report Server:
```

Comment:
administrator = implementer, according to our docs.

Shouldn't it be more correct to say "Following the recommendations in this section is a responsibility of the implementer who deploys and configures the TRS. These recommendations pertain to general security settings for web applications deployed on Windows IIS Servers. Following the principle "safe-by-default", the product installer and the initial setup workflow will configure Telerik Report Server using the safest options where possible. All of the items listed below can be configured through the web application's registration in IIS, or through the product's Configuration page: